fedimint-cli/src/client.rs

```diff
@@ -118,6 +120,10 @@ pub enum ClientCmd {
     },
     /// Print the secret key of the client
     PrintSecret,
+    ListOperations {
```

Comment:
Is this for currently active / all current and past? Would it make sense to add a some filters right away?

Reply:
I'd rather compose queries via `jq`. This is not a user-facing tool.